src/dialog/reimu.c

```diff
@@ -734,67 +734,67 @@ DIALOG_TASK(reimu, StageExPreBoss) {
+	MSG(yumemi, "Do you see this place we're in, shrine maiden? Do you even know how it was made?");
+	MSG(yumemi, "Whether you believe it or not, this machine should have been able to save the world.");
 	FACE(yumemi, eyes_closed);
-	MSG(yumemi, "But at least, with this machine I’ve built, I can erase every so-called ‘Otherworld’ from the fabric of social reality.");
-	FACE(yumemi, normal);
+	MSG(yumemi, "But even when I revealed this technology to the world, nobody woke up from their dissociative haze.");
+	MSG(yumemi, "Nothing changed. And I figured out why.");
+	MSG(yumemi, "It's because of places like this. Why work on the real world when you can pretend to in another?");
 
 	FACE(reimu, assertive);
+	FACE(yumemi, normal);
 	MSG(reimu, "People won’t stop day-dreaming just because you tell them to. Fantasy is in people’s hearts and minds.");
-	FACE(reimu, sigh);
-	MSG(reimu, "So, even if you managed to destroy Gensōkyō—");
 
 	MSG(yumemi, "Yōkai, Gods, magicians, flying shrine maidens…");
 	FACE(yumemi, eyes_closed);
-	MSG(yumemi, "It’s all delusional nonsense.");
+	MSG(yumemi, "It’s all delusional nonsense. And I now have the power to erase it all.");
 
 	FACE(reimu, unsettled);
 	MSG(reimu, "So you want genocide?");
 
 	FACE(yumemi, surprised);
-	MSG(yumemi, "Genocide…? What a harsh word.");
+	MSG(yumemi, "Genocide…? What a nasty word.");
 	FACE(yumemi, normal);
 	FACE(reimu, unamused);
 	MSG(yumemi, "You’re not real. No one is being killed.");
 
 	MSG(reimu, "No wonder those two got freaked out just thinking about you. You’re completely unreasonable.");
+	FACE(reimu, sigh);
+	MSG(reimu, "Anyway, even if you managed to destroy Gensōkyō—");
 
 	FACE(yumemi, eyes_closed);
 	MSG(yumemi, "Ah, I thought I’d made them both see things my way, by introducing them to the Grand Unified Theory.");
 	FACE(yumemi, normal);
+	FACE(reimu, irritated);
 	MSG(yumemi, "But in the end, they just betrayed me.");
 	FACE(yumemi, sad);
 	MSG(yumemi, "In hindsight, it makes sense, of course. They can’t betray what they are.");
 
-	FACE(reimu, irritated);
 	MSG(reimu, "It’s hard to keep people on your side when all you do is threaten them, idiot!");
 	FACE(reimu, assertive);
 	MSG(reimu, "Have you tried not being a genocidal maniac?");
 
 	FACE(yumemi, sigh);
 	MSG(yumemi, "I’m tired, shrine maiden. Too tired to explain my life to a figment of imagination.");
 	FACE(reimu, unamused);
-	FACE(yumemi, eyes_closed);
-	MSG(yumemi, "Perhaps I’ve already given up trying to save my society, too.");
 
 	FACE(reimu, assertive);
 	MSG(reimu, "Sucks to be you. Don’t make it our problem.");
 
 	EVENT(music_changes);
 
 	FACE(yumemi, normal);
-	MSG(yumemi, "Oh, but it’s inherently your problem, isn’t it? Your very existence is the root of my misery.");
+	MSG(yumemi, "Oh, but it’s inherently your problem, isn’t it? Your existence does nothing but hold humanity back.");
```

Comment:
I think I prefer the old line here as well... Doesn't seem like she'd have much care left for humanity at this point; she even admits that this whole thing is probably futile. The way I see it she's taking her frustration out on Gensokyo more than actually trying to fix anything. Plus the loss of Merry gives more context to "Your very existence is the root of my misery".